<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorPage</class>
 <widget class="QWidget" name="AdministratorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/patient.svg</normaloff>:/Forms/Icons/Forms/Icons/patient.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Пациенты</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="PatientTableView"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="AddPatientButton">
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/add.svg</normaloff>:/Forms/Icons/Forms/Icons/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="EditPatientButton">
         <property name="text">
          <string>Редактировать</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/edit.svg</normaloff>:/Forms/Icons/Forms/Icons/edit.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="RemovePatientButton">
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/remove.svg</normaloff>:/Forms/Icons/Forms/Icons/remove.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/registrator.svg</normaloff>:/Forms/Icons/Forms/Icons/registrator.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Регистраторы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="RegistratorTableView"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="AddRegistratorButton">
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/add.svg</normaloff>:/Forms/Icons/Forms/Icons/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="EditRegistratorButton">
         <property name="text">
          <string>Редактировать</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/edit.svg</normaloff>:/Forms/Icons/Forms/Icons/edit.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="RemoveRegistratorButton">
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/remove.svg</normaloff>:/Forms/Icons/Forms/Icons/remove.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/admin.svg</normaloff>:/Forms/Icons/Forms/Icons/admin.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Врачи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="DoctorsTableView"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="AddDoctorButton">
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/add.svg</normaloff>:/Forms/Icons/Forms/Icons/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="EditDoctorButton">
         <property name="text">
          <string>Редактировать</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/edit.svg</normaloff>:/Forms/Icons/Forms/Icons/edit.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="RemoveDoctorButton">
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/remove.svg</normaloff>:/Forms/Icons/Forms/Icons/remove.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/time.svg</normaloff>:/Forms/Icons/Forms/Icons/time.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Записи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="AppointmentTableView"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="RemoveAppointmentButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/remove.svg</normaloff>:/Forms/Icons/Forms/Icons/remove.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="AddAppointmentButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Добавить запись</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/add.svg</normaloff>:/Forms/Icons/Forms/Icons/add.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/card.svg</normaloff>:/Forms/Icons/Forms/Icons/card.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Карты</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="5" column="0">
        <widget class="QPushButton" name="AddCardRecordButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/add.svg</normaloff>:/Forms/Icons/Forms/Icons/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="RemoveCardRecordButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/remove.svg</normaloff>:/Forms/Icons/Forms/Icons/remove.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ФИО пациента</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="PatientFullNameField"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="FindCardButton">
           <property name="text">
            <string>Открыть карту</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTableWidget" name="CardRecordTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
